src/worker/sandbox/cgroups/cgroup.go

```diff
@@ -43,7 +37,9 @@ func (cg *CgroupImpl) Release() {
 				if i == 0 {
 					panic(fmt.Errorf("Cannot release cgroup that contains processes: %v", pids))
 				} else {
-					cg.printf("cgroup Rmdir failed, trying again in 5ms")
+					if common.Conf.Trace.Cgroups {
```

Comment:
do we need to check this each time?  Can't we just have a logger that somehow drops the log messages if it's not enabled?